frontend/src/app/buyer/order/[id]/page.tsx

```diff
@@ -94,6 +169,87 @@ export default async function OrderPage({
       </div>
 
       <div className="grid gap-6">
+
+        {/* Payment Banner */}
+        <Card className={`${bannerStyle.borderColor}`}>
+          <CardHeader className="py-6">
+            <CardTitle className={bannerStyle.textColor}>
+              {bannerStyle.title}
+            </CardTitle>
+            <CardDescription className={bannerStyle.textColor}>
+              {bannerStyle.message}
+            </CardDescription>
+          </CardHeader>
+           {/* Show CardContent for PENDING VENMO orders */}
+           {order.paymentStatus === "PENDING" && order.paymentMethod === "VENMO" && (
+            <CardContent>
+              <div className="space-y-4">
+                <div className="flex justify-center">
+                  <Button
+                    size="lg"
+                    asChild
+                    className="flex items-center gap-2 bg-[#3D95CE] hover:bg-[#2E7BB8] text-white font-semibold px-8 py-3 text-md"
+                  >
+                    <a
+                      href={fundraiser.venmoUsername
+                        ? `https://venmo.com/${fundraiser.venmoUsername}?txn=pay&note=${encodeURIComponent(orderIdForPayment)}&amount=${encodeURIComponent(orderTotal)}`
+                        : `https://venmo.com?txn=pay&note=${encodeURIComponent(orderIdForPayment)}&amount=${encodeURIComponent(orderTotal)}`
```

Comment:
I'm not sure we should still have the venmo payment link if seller venmo username is not filled. If the seller didn't put their venmo username, then how will buyer know who to pay?